packages/cli/src/options/common.js

```diff
@@ -16,7 +16,7 @@ export default {
     alias: 'c',
     type: 'string',
     default: defaultNuxtConfigFile,
-    description: `Path to Nuxt.js config file (default: \`${defaultNuxtConfigFile}\`)`
+    description: `Path to Nuxt.js config file (default: \`${defaultNuxtConfigFile}.js\`)`
```

Comment:
Why `.js` added here?

Reply:
Cause it will say `Path to Nuxt.js config file (default: nuxt.config)`, do we care not having `.js` then ?

Reply:
@pi0

Reply:
Well, if we are shipping `*.ts` support somehow into the core it may worth doing this to be extensionless.

Reply:
Alright, updated, thanks :)